WooCommerce/Classes/ViewRelated/Orders/Payment Methods/PaymentMethodsViewModel.swift

```diff
@@ -148,7 +148,7 @@ final class PaymentMethodsViewModel: ObservableObject {
          paymentLink: URL? = nil,
          formattedTotal: String,
          flow: WooAnalyticsEvent.PaymentsFlow.Flow,
-         isTapToPayOnIPhoneEnabled: Bool = ServiceLocator.generalAppSettings.settings.isTapToPayOnIPhoneSwitchEnabled,
+         isTapToPayOnIPhoneEnabled: Bool = true,
```

Comment:
Do we need this flag now that it's true?

Reply:
No, but it's left in for ease of the revert if we do that in the beta week, as discussed on P2. I'll take it out when we release 12.6